src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -157,9 +156,6 @@ export default class DefaultAudioVideoController
     this._audioMixController = new DefaultAudioMixController(this._logger);
     this.meetingSessionContext.logger = this._logger;
     this._eventController = new DefaultEventController(this, eventReporter);
-    if (configuration.videoDownlinkBandwidthPolicy instanceof VideoPriorityBasedPolicy) {
```

Comment:
Does `instanceof` not work for subclasses?

Reply:
No it works for subclasses too but it did not work if someone implements `VideoDownlinkBandwidthPolicy` and wants to have access to tileController. So to fix this issue with Simulcast, I could just do an `instanceof` check here but I just expose the method to be more general.